<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWidget</class>
 <widget class="QWidget" name="TestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_1">
      <property name="title">
       <string>Tabla de Test de Aleatoriedad</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Tabla de pruebas:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Muestra en un ambiente amigable&lt;/p&gt;&lt;p&gt;los números probar.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::TargetMoveAction</enum>
         </property>
         <column>
          <property name="text">
           <string>#</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cantidad de números a probar.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Comprobar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Test de Aleatoriedad</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>139</width>
             <height>335</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Uniformidad</string>
           </attribute>
           <attribute name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Selector de pruebas:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Muestra en un ambiente amigable&lt;/p&gt;&lt;p&gt;las pruebas de aleatoriedad.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QToolBox" name="toolBox_2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="page_3">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>172</width>
                 <height>246</height>
                </rect>
               </property>
               <attribute name="label">
                <string>Kolmoworrow - Smirrox</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Nivel de significación</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox">
                    <item>
                     <property name="text">
                      <string>0.10</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>0.05</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>0.01</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_4">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>178</width>
                 <height>246</height>
                </rect>
               </property>
               <attribute name="label">
                <string>Chi - Cuadrado</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Nivel de significación</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox_2">
                    <item>
                     <property name="text">
                      <string>0.005</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>0.01</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>0.025</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>0.05</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>0.1</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Números de clases</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_2">
                    <property name="minimum">
                     <number>5</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>139</width>
             <height>335</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Independencia</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QToolBox" name="toolBox_3">
              <property name="currentIndex">
               <number>3</number>
              </property>
              <widget class="QWidget" name="page_8">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>166</width>
                 <height>192</height>
                </rect>
               </property>
               <attribute name="label">
                <string>Ascendentes y Descendentes</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Nivel de significación</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_9">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>166</width>
                 <height>192</height>
                </rect>
               </property>
               <attribute name="label">
                <string>Encima y Debajo de la Media</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Nivel de significación</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_10">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>159</width>
                 <height>192</height>
                </rect>
               </property>
               <attribute name="label">
                <string>Longitud de Corridas</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <widget class="QComboBox" name="comboBox_3">
                  <item>
                   <property name="text">
                    <string>Ascendete y Descendete</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Encima y Debajo de la Media</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Nivel de signififació</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_5"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_11">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>159</width>
                 <height>192</height>
                </rect>
               </property>
               <attribute name="label">
                <string>Actocorelacción</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_15">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Nivel de signififació</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_6"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>i =</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_3"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>m =</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_4"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Resultados</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Documentación</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Vista rapida:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Muestra en un ambiente amigable&lt;/p&gt;&lt;p&gt;algunos conceptos importantes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
